### PR TITLE
Add Polestar

### LINKED ISSUE #9513: Add Polestar (Car Brand) icon
### Brand Name

Polestar

### Website

https://www.polestar.com/

### Popularity Metric

The Similarweb rank is 18915 https://www.similarweb.com/website/polestar.com/#overview

### Official Resources for Icon and Color

https://en.wikipedia.org/wiki/File:Polestar_Logo.svg

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
polestar

**Issue:** closes #9513 

**Similarweb rank:** 18,915. See https://www.similarweb.com/website/polestar.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://www.polestar.com/w3-assets/safari-pinned-tab.svg
Color: Taken from logo `#000000`

### CODE REVIEW

**Comment by dgomes** [comment]
Thank you soo much for your work :)

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10207,6 +10207,12 @@
             "hex": "CD792C",
             "source": "https://pola.rs"
         },
+        {
+            "title": "Polestar",
+            "hex": "000000",
+            "source": "https://www.polestar.com/w3-assets/safari-pinned-tab.svg",
+            "guidelines": "https://media.polestar.com"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:10214]:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Polestar_logo_2020.svg"
```

File: icons/polestar.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Polestar</title><path d="M11.911 17.931c0 5.46-.01 5.688-.188 5.906-.158.188-.208.208-.247.089-.02-.08-.455-2.196-.95-4.699s-.94-4.57-.989-4.6c-.04-.03-2.176-.475-4.738-.99l-4.65-.939.208-.218.208-.227H11.91v5.678zm11.742-6.41-.277.237-5.609-.02-5.609-.03-.03-5.618c-.019-5.233-.01-5.639.15-5.856.217-.307.306-.307.375-.03.03.129.455 2.226.94 4.67.485 2.443.9 4.45.91 4.47.02.01 2.077.426 4.58.93 2.503.495 4.62.92 4.699.95.119.04.089.109-.129.297z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/polestar.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Polestar</title><path d="m12.549 0-.457.555v11.191h11.19l.554-.457-9.4-1.89zM.719 12.26l-.555.457L9.563 14.6l1.886 9.4.457-.555V12.26Z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Review by adamrusted — APPROVED** [review]

**Comment by dgomes** [comment]
🎉🙏

**Pull request merged**